Formula/grpc.rb

```diff
@@ -75,6 +88,15 @@ def install
   end
 
   test do
+    # `brew` has a bug that breaks `ENV.cc` in test blocks when
+    # compiling with brewed LLVM. The obvious workaround doesn't
+    # work because of an audit that prevents setting CC=clang.
+    # TODO: Fix this terrible hack.
+    on_macos do
+      compiler = "clang"
+      ENV["CC"] = compiler
+    end
```

Comment:
This should be fixed properly.

At this point I'd argue that if the goal is to select a specific compiler rather than reject others, then simply setting `HOMEBREW_CC` in the `install` block is probably easier.

Though that's not really documented and there should probably be a proper DSL for it.

Reply:
Setting `HOMEBREW_CC` didn't work, unfortunately. Either that or I did it wrong.

Reply:
I just realised `ENV.llvm_clang` exists. Maybe try call that instead? It'll set `HOMEBREW_CC` for you.